exercises/say/uSayTest.pas

```diff
@@ -0,0 +1,106 @@
+unit uSayTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+const
+  CanonicalVersion = '1.2.0';
+
+type
+  [TestFixture]
+  TSayTest = class(TObject)
+  private
+    procedure Setup;
+    procedure TearDown;
+  public
+    [Test]
+//    [Ignore('Comment the "[Ignore]" statement to run the test')]
+    [TestCase('Test zero','0,zero')]
+    [TestCase('Test One','1,one')]
+    procedure TestDigits(const ANum : int64; const AWord : string );
```

Comment:
I think we will need to change this.  `[Ignore]` works on the entire test, not individual test cases.  To "skip" a test case the `TestCase` statement must be commented out.  This would be introducing a new way of (in/ex)cluding test cases.

Your use of the `TestCase` is spot on. :+1: 

FYI when using `[TestCase()]` `[Test]` becomes optional.

If we want to stick with what you have done you will need to introduce some comment that takes up the role of the comment on line 18.  Also, `[Ignore]` won't be needed.  If all testcases above a procedure are commented then that test case will not be called at runtime, effectively ignored.